src/components/ui/Button/Button.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react'
+import PropTypes from "prop-types";
+import './Button.css'
+
+const  TFButton = (props) => {
+    const {icon, label, disabled, variant, handleClick, size, customStyles} = props;
```

Comment:
instead of customStyles use the prop name styl;es so we can maintain consistency